### PR TITLE
Fix: empty help strings

### PR DESCRIPTION
**Pull request by DeviaVir**
This fixes:
``` 
# curl localhost:9104/metrics
An error has occurred during metrics collection:

62 error(s) occurred:
* empty help string
* empty help string
* empty help string
* empty help string
* [...times 62...]
```
and no metrics being returned at all.

I know some of these don't make sense, but at least it made the aurora exporter work again for me.

### CODE REVIEW

**Comment by DeviaVir** [comment]
@tommyulfsparre ping?

File: vars.go [hunk 1]
```diff
@@ -40,10 +41,12 @@ var counters = map[string]*prometheus.Desc{
 		"Lost garbage collection task total.",
 	),
 	"http_200_responses_events": newDesc(
-		"http_200", "responses_events", "",
+		"http_200", "responses_events",
+		"Number of 200OK response events.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Number of HTTP 200 status response events.",
```

File: vars.go [hunk 2]
```diff
@@ -40,10 +41,12 @@ var counters = map[string]*prometheus.Desc{
...
 	),
 	"http_200_responses_nanos_total": newDesc(
-		"http_200", "responses_nanos_total", "",
+		"http_200", "responses_nanos_total",
+		"Timed 200OK response total.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Timed HTTP 200 status response total.",
```

File: vars.go [hunk 3]
```diff
@@ -323,13 +374,16 @@ var counters = map[string]*prometheus.Desc{
 
 var gauges = map[string]*prometheus.Desc{
 	"http_200_responses_events_per_sec": newDesc(
-		"http_200", "responses_events_per_sec", "",
+		"http_200", "responses_events_per_sec",
+		"Number of 200OK response events per second.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Number of HTTP 200 status response events per second.",
```

File: vars.go [hunk 4]
```diff
@@ -323,13 +374,16 @@ var counters = map[string]*prometheus.Desc{
...
 	),
 	"http_200_responses_nanos_per_event": newDesc(
-		"http_200", "responses_nanos_per_event", "",
+		"http_200", "responses_nanos_per_event",
+		"Timed 200OK responses per event.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Timed HTTP 200 status responses per event.",
```

File: vars.go [hunk 5]
```diff
@@ -323,13 +374,16 @@ var counters = map[string]*prometheus.Desc{
...
 	),
 	"http_200_responses_nanos_total_per_sec": newDesc(
-		"http_200", "responses_nanos_total_per_sec", "",
+		"http_200", "responses_nanos_total_per_sec",
+		"Number of 200OK response events total per second.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Number of HTTP 200 status response events total per second.",
```

File: vars.go [hunk 6]
```diff
@@ -389,62 +446,80 @@ var gauges = map[string]*prometheus.Desc{
 		"CPU time used by the process.",
 	),
 	"pubsub_executor_queue_size": newDesc(
-		"", "pubsub_executor_queue_size", "",
+		"", "pubsub_executor_queue_size",
+		"Number of queue size.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Current pubsub executor queue size.",
```

File: vars.go [hunk 7]
```diff
@@ -389,62 +446,80 @@ var gauges = map[string]*prometheus.Desc{
...
 	),
 	"scheduled_task_penalty_events_per_sec": newDesc(
-		"scheduled_task", "penalty_events_per_sec", "",
+		"scheduled_task", "penalty_events_per_sec",
+		"Number of penalty events per second.",
 	),
 	"scheduled_task_penalty_ms_per_event": newDesc(
-		"scheduled_task", "penalty_ms_per_event", "",
+		"scheduled_task", "penalty_ms_per_event",
+		"NUmber of penalty MS per event.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Number of scheduled task penalty in millisecond per event.",
```

File: vars.go [hunk 8]
```diff
@@ -389,62 +446,80 @@ var gauges = map[string]*prometheus.Desc{
...
 	),
 	"scheduled_task_penalty_ms_total_per_sec": newDesc(
-		"scheduled_task", "penalty_ms_total_per_sec", "",
+		"scheduled_task", "penalty_ms_total_per_sec",
+		"Number of penalty MS total per second.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Number of scheduled task penalty in millisecond total per event.",
```

File: vars.go [hunk 9]
```diff
@@ -177,10 +195,12 @@ var counters = map[string]*prometheus.Desc{
...
 	),
 	"scheduled_task_penalty_events": newDesc(
-		"scheduled_task", "penalty_events", "",
+		"scheduled_task", "penalty_events",
+		"Number of tasks with penalty events.",
 	),
 	"scheduled_task_penalty_ms_total": newDesc(
-		"scheduled_task", "penalty_ms_total", "",
+		"scheduled_task", "penalty_ms_total",
+		"TImed penalty events total.",
```
Comment (tommyulfsparre) [inline vars.go]:
```suggestion
		"Timed penalty events total.",
```

**Review by tommyulfsparre — CHANGES_REQUESTED** [review]

**Comment by tommyulfsparre** [comment]
@DeviaVir Thanks! 
I couldn't reproduce this myself with `prometheus/client_golang` version 0.8.0 and 0.9.0. Which version did you build against?

**Comment by DeviaVir** [comment]
@tommyulfsparre 
> I couldn't reproduce this myself with prometheus/client_golang version 0.8.0 and 0.9.0. Which version did you build against?

here's a dockerfile to reproduce against:
```
FROM library/golang:latest # 1.11.1 at this time
RUN go get github.com/tommyulfsparre/aurora_exporter \
  && go build github.com/tommyulfsparre/aurora_exporter
```
then using that aurora_exporter and trying to curl `/metrics` ended up in the error in OP

**Comment by DeviaVir** [comment]
@tommyulfsparre updated the PR to include your requested changes.

**Review by tommyulfsparre — APPROVED** [review]

**Comment by tommyulfsparre** [comment]
@DeviaVir thanks!

**Pull request merged**